### PR TITLE
Umcs 123/change/documentation links

### PR DESCRIPTION
**Pull request by Ryouzanpaku**

### CODE REVIEW

**Review by sixer1182 — COMMENTED** [review]
What about the Changelog?
Can you please add all changes which are done in comparison to the old plugin?

**Review by sixer1182 — COMMENTED** [review]
Changelog update?

File: etc/config.xml [hunk 1]
```diff
@@ -3,8 +3,8 @@
         <payment>
             <unzer>
                 <active>0</active>
-                <public_key>s-pub-2a10ifVINFAjpQJ9qW8jBe5OJPBx6Gxa</public_key>
-                <private_key>s-priv-2a102ZMq3gV4I3zJ888J7RR6u75oqK3n</private_key>
+                <public_key></public_key>
+                <private_key></private_key>
```
Comment (sixer1182) [inline etc/config.xml:7]: This does not match the commit message

Comment (Ryouzanpaku) [reply etc/config.xml:7]: - Commit message adresses this change. 
-  Changelog is updated now.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [1.0.0][1.0.0]
 ### Changed
-*   Rebranding of the Plugin.
+* Rebranding of the Plugin.
+* Remove preconfigured test keypair from config.
```
Comment (sixer1182) [inline CHANGELOG.md:9]: SDK update is missing here

Comment (Ryouzanpaku) [reply CHANGELOG.md:9]: Done.

**Review by sixer1182 — APPROVED** [review]
LGTM

**Pull request merged**